ocpmodels/preprocessing/atoms_to_graphs.py

```diff
@@ -162,7 +162,12 @@ def convert(
             data.distances = edge_distances
         if self.r_fixed:
             fixed_idx = torch.zeros(natoms)
-            fixed_idx[atoms.constraints[0].index] = 1
+            if hasattr(atoms, "constraints"):
+                from ase.constraints import FixAtoms
```

Comment:
very minor nit: import could be done at the top of the script with other imports, it's usually more clear that way.